apis/projectcontour/v1/httpproxy.go

```diff
@@ -54,14 +54,14 @@ type Include struct {
 	// structs.
 	// When applied, they are merged using AND, with one exception:
 	// There can be only one Prefix MatchCondition per Conditions slice.
-	// More than one Prefix, or contradictory Conditions, will make the
+	// More than one Prefix or Regex, or contradictory Conditions, will make the
```

Comment:
```suggestion
	// More than one Prefix, or contradictory Conditions, will make the
```

Alternately, you could add some text here also describing that exact & regex are not supported at all for an `Include`, but that's already covered on the fields in `MatchCondition` below so I don't think it's strictly necessary.